lxd/devices.go

```diff
@@ -817,7 +817,7 @@ func deviceUSBEvent(s *state.State, usb usbDevice) {
 func deviceEventListener(s *state.State) {
 	chNetlinkCPU, chNetlinkNetwork, chUSB, err := deviceNetlinkListener()
 	if err != nil {
-		logger.Errorf("scheduler: couldn't setup netlink listener")
+		logger.Errorf("scheduler: couldn't setup netlink listener: %v", err)
```

Comment:
nit and not necessary but probably be clearer if: "Couldn't set up netlink listener".

Reply:
There's all wrongly capitalized so I left it for later.